app/inbound/handle-event/external-event.js

```diff
@@ -0,0 +1,14 @@
+const { createRow } = require('./create-row')
+const { EXTERNAL_EVENT } = require('../../constants/event-types')
+const { getClient } = require('../../storage')
+const { createIfNotExists } = require('./create-if-not-exists')
+const saveExternalEvent = async (event) => {
+  const entity = createRow(event.partitionKey ?? event.id, event.id, EXTERNAL_EVENT, event)
```

Comment:
We will need to enhance this to save in mutliple ways depending on the event type

Reply:
have added more coverage for this